build-common.xml

```diff
@@ -1850,7 +1850,7 @@ artifact.url=${remote.repository.cdn.url}/com/liferay/portal/com.liferay.${proce
 				<path refid="lib.pre.classpath" />
 				<fileset
 					dir="${project.dir}/lib/development"
-					excludes="osgi-annotation-versioning.jar,jalopy.jar,jetty*.jar,tomcat*.jar"
+					excludes="jalopy.jar,jetty*.jar,tomcat*.jar"
```

Comment:
@dantewang @yuhai Why this is causing unit test compile faile? why we did not see it on log4j1?